lib/auth/grpcserver.go

```diff
@@ -5141,6 +5143,17 @@ func NewGRPCServer(cfg GRPCServerConfig) (*GRPCServer, error) {
 	collectortracepb.RegisterTraceServiceServer(server, authServer)
 	auditlogpb.RegisterAuditLogServiceServer(server, authServer)
 
+	dynamicWindows, err := dynamicwindowsv1.NewService(dynamicwindowsv1.ServiceConfig{
+		Authorizer: cfg.Authorizer,
+		Backend:    cfg.AuthServer.Services,
+		// TODO(probakowski): switch to cache when support is added
```

Comment:
Is this TODO still relevant?

Reply:
Yes, it will go away in https://github.com/gravitational/teleport/pull/46989